src/player.cpp

```diff
@@ -2941,11 +2941,15 @@ void player::shout( std::string msg )
     }
 
     if( noise <= minimum_noise ) {
-        add_msg( m_warning, _( "The sound of your voice is almost completely muffled!" ) );
+        add_msg_player_or_npc( m_warning,
```

Comment:
No need to print this one if a NPC is shouting, the player can't hear it anyway.

Reply:
`noise <= minimum_noise`
NPC's shout could have volume 2, and if it is, player can hear it, right?